util/src/test/java/com/netcracker/ConsoleHelperTest.java

```diff
@@ -0,0 +1,89 @@
+package com.netcracker;
+
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.TextFromStandardInputStream;
+import org.junit.rules.ExpectedException;
+
+import java.io.*;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Modifier;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.contrib.java.lang.system.TextFromStandardInputStream.emptyStandardInputStream;
+
+public class ConsoleHelperTest {
+    private static final PrintStream SYSTEM_OUT = System.out;
+    @Rule
+    public final TextFromStandardInputStream systemIn = emptyStandardInputStream();
+    @Rule
+    public final ExpectedException exception = ExpectedException.none();
+    private OutputStream resultMessageStream;
+
+    @Before
+    public void setUpEachTest() {
+        resultMessageStream = new ByteArrayOutputStream();
+        PrintStream outputStream = new PrintStream(resultMessageStream);
+        System.setOut(outputStream);
+    }
+
+    @Before
+    public void reloadReaderInConsoleHelperClass() throws NoSuchFieldException, IllegalAccessException {
+        Field bufferedReader = ConsoleHelper.class.getDeclaredField("READER");
+        bufferedReader.setAccessible(true);
+        Field modifiers = bufferedReader.getClass().getDeclaredField("modifiers");
+        modifiers.setAccessible(true);
+        modifiers.setInt(bufferedReader, bufferedReader.getModifiers() & ~Modifier.FINAL);
```

Comment:
It reload READER in ConsoleHelper class, so this variable can see all changes, I try to do something without it, but when I try to simulate user input it returns null, when I try to readString()